README.md

```diff
@@ -1084,6 +1084,12 @@ This device does not have native trunking. It does not support the following att
 * ensure
 * encapsulation
 
+##### 3650 
+
+##### 4503
+
+This device only supports a single method of encapsulation, `802.1q`, and as such the attribute to set it is not supported.
```

Comment:
Any links to docs to support this statement? And it should be under both `3650` and `4503` if it applies to both

Reply:
I have found several mentions of it in discussion threads, but i apparently has not been updated in the actual docs. 
https://community.cisco.com/t5/switching/quot-switchport-trunk-encapsulation-quot-not-supported-in/td-p/2734528
https://community.cisco.com/t5/switching/c3020-blade-switch-doesn-t-accepting-dot1q-on-trunk-link/td-p/2677079

Reply:
Can the list here mirror the command.yaml?

Readme:
- excludes 3650 and 4503

Command.yaml:
- excludes 2960 and XE OS

Reply:
updated